pkg/provider/azure.go

```diff
@@ -488,6 +501,8 @@ func (az *Cloud) InitializeCloudFromConfig(config *Config, fromSecret bool) erro
 	az.routeUpdater = newDelayedRouteUpdater(az, routeUpdateInterval)
 	go az.routeUpdater.run()
 
+	go az.refreshZones(az.getRegionZonesMap)
```

Comment:
rename `getRegionZonesMap` to `syncRegionZonesMap`

Reply:
done